store/transfer_processor.go

```diff
@@ -88,12 +84,14 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 			addrs = append(addrs, tx.From)
 		}
 	}
-	// From transfer events
+	// From transfer events (including miner and uncles)
 	for _, e := range events {
 		fromAddr := gethCommon.BytesToAddress(e.From)
 		if _, ok := seenAddrs[fromAddr]; !ok {
-			seenAddrs[fromAddr] = struct{}{}
-			addrs = append(addrs, e.From)
+			if !e.IsMinerRewardEvent() && !e.IsUncleRewardEvent() {
```

Comment:
Suggest
```
if e.IsMinerRewardEvent() || e.IsUncleRewardEvent() {
    continue
}
fromAddr := gethCommon.BytesToAddress(e.From)
...
```

Reply:
ok, but the `if e.IsMinerRewardEvent() || e.IsUncleRewardEvent()` condition expression should be in  `if _, ok := seenAddrs[fromAddr]; !ok`.

Reply:
oh...I couldn't use `continue` keyword. if expression is satisfied, we will miss `toAddr`.